### ISSUE TITLE
I will start editing again once the English text has caught up.

### ISSUE THREAD

**Issue by Tanish-Eagle** [origin-brain/rust-by-practice#379]
I will restart editing once the English text has caught up. Currently, I am waiting for more text to appear for smart pointers and beyond.

**Comment by stappersg** [comment]
Those we can close this issue, please close it.

Because dangling issues do drain human energy.

**Comment by Tanish-Eagle** [comment]
Very well, I will close this issue. I was waiting from some update from the project, but it has not come yet.

**Comment by stappersg** [comment]
On Sat, Aug 05, 2023 at 09:45:18AM -0700, Tanish-Eagle wrote:
> ... I was waiting from some update from the project, but it has not come yet.

Advice: State clearly what update _you_ need.

Preferable in an open (or re-opened) issue.
And take ownership of it.


Groeten
Geert Stappers
Has no idea what is blocking Tanish-Eagle
and want to go beyond it, to go a next challenge.
-- 
Silence is hard to parse

**Comment by Tanish-Eagle** [comment]
I was waiting for more text to appear on the smart pointers and beyond, but currently there is nothing there. As a result, I could not perform my editing duties. That is what blocking me.

**Comment by stappersg** [comment]
On Sun, Aug 06, 2023 at 11:28:01AM -0700, Tanish-Eagle wrote:
> I was waiting for more text to appear on the smart pointers and beyond,
> but currently there is nothing there. As a result, I could not perform
> my editing duties. That is what blocking me.

Acknowlege.
Thanks for closing the issue.
Thanks for explaining what is needed at your side.

For what it is worth:
Doing is much better as talking about what will be done.
 

Groeten
Geert Stappers
-- 
Silence is hard to parse
